model_analyzer/config/generate/optuna_run_config_generator.py

```diff
@@ -451,6 +455,44 @@ def _create_objective_based_run_config(
         return run_config
 
     def _create_parameter_combo(
+        self,
+        model: ModelProfileSpec,
+        trial_objectives: TrialObjectives,
+        composing_trial_objectives: ComposingTrialObjectives,
+    ) -> ParameterCombo:
+        if model.is_ensemble():
+            param_combo = self._create_ensemble_parameter_combo(
+                composing_trial_objectives
+            )
+        else:
+            param_combo = self._create_non_ensemble_parameter_combo(trial_objectives)
+
+        return param_combo
+
+    def _create_ensemble_parameter_combo(
+        self,
+        composing_trial_objectives: ComposingTrialObjectives,
+    ) -> ParameterCombo:
+        """
+        For the ensemble model the only parameter we need to set
```

Comment:
I just want to double check how the parent vs composing models are handled. This function is only called for the parent model, right? And the composing models are always handled elsewhere?

Reply:
That's correct.